src/styles/_catalog.scss

```diff
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
   max-width: 944px;
   margin: 0 auto;
```

Comment:
delete this, and let grid dictate width of container, also delete margins and find better way to place them